### PR TITLE
routes for spaces added

### PR DESCRIPTION
**Pull request by iTTUD-Pingouin**
J'ai ajouté les routes relatives au controller des SPACES

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   devise_for :users
 
+  resources :spaces
```
Comment (virgilepradel) [inline config/routes.rb:4]: looks like you added all possible routes ! cool !

**Review by virgilepradel — APPROVED** [review]
I am Virgile Pradel and I approve this pull request

File: config/routes.rb (hunk 1 above)
Comment (iTTUD-Pingouin) [reply config/routes.rb:4]: Ok can you merge ?

Comment (virgilepradel) [reply config/routes.rb:4]: believe in yourself. Let's do it

Comment (Matz3470) [reply config/routes.rb:4]: Je suis d'accord

Comment (Matz3470) [reply config/routes.rb:4]: well done guys Im proud of you

**Pull request merged**